src/global/completions.rs

```diff
@@ -241,3 +249,13 @@ pub(crate) async fn completions_sync_status(
 
     Ok((completions_to_remove, completions_to_add))
 }
+
+#[cfg(not(unix))]
+pub(crate) async fn completions_sync_status(
```

Comment:
maybe we can add a small comment ( from the PR overview ) to mention that we never install completions on windows that's why we don't need to check it here?

Reply:
Done